proposals/lip-0042.md

```diff
@@ -127,6 +127,10 @@ The Reward module has the following internal function.
 
 This function is used to retrieve the reward of a block at a given height.
 
+##### Parameters
+* `blockHeader`: header of a block.
+* `blockAssets`: a block property as described in [LIP 55][lip-0055#block-assets].
```

Comment:
I think it make sense to go for the first choice (which is essentially what Vikas does here), since we are in the reward module and we just have access to the block, so it make sense to me to leave the decoding and the search for the particular asset to the random module. But not really strong opinion here.

Reply:
Agree with Greg but we can discuss it.

Reply:
I checked the sdk code, they are getting it as below
asset = blockAssets.getAsset(this._moduleName);

I assume our LIP description is sufficient.